web/case/views.py

```diff
@@ -126,6 +138,26 @@ def add_verification_details(case_id):
     except Exception as e:
         return render_template("error.html", error_msg=str(e))
 
+@blueprint.route("/cases/<case_id>/add_vote_to_case", methods = ["GET", "POST"])
+def add_vote_to_case(case_id):
+    try:
+        if request.method == "GET":
```

Comment:
as discussed in the call , remove this GET portion as its not needed.

Reply:
without GET portion how is it possible to go to case voting page.

Reply:
ok i see .. then np.  you can keep

Reply:
ok